sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -221,6 +243,8 @@ public static Write write() {
 
       abstract Builder setTableId(String tableId);
 
+      abstract Builder setLocalhost(String host);
```

Comment:
Please drop this and just leave appropriate documentation for how to configure the `CloudBigtableOptions` object.